---
{"_attributes":{"version":"4.4.4+","urlbase":"https://bugs.ecmascript.org/","maintainer":"dherman@mozilla.com"},"bug":{"bug_id":2641,"creation_ts":"2014-04-11 11:22:00 -0700","short_desc":"9.4.4.4 [[Set]]: Exotic arguments [[Set]] does not check receiver","delta_ts":"2014-04-29 21:52:38 -0700","product":"Draft for 6th Edition","component":"technical issue","version":"Rev 23: April 5, 2014 Draft","rep_platform":"All","op_sys":"All","bug_status":"RESOLVED","resolution":"FIXED","priority":"Normal","bug_severity":"normal","everconfirmed":true,"reporter":{"uid":"andrebargull","name":"André Bargull"},"assigned_to":{"uid":"allen","name":"Allen Wirfs-Brock"},"long_desc":[{"commentid":7673,"comment_count":0,"who":{"uid":"andrebargull","name":"André Bargull"},"bug_when":"2014-04-11 11:22:54 -0700","thetext":"9.4.4.4 [[Set]] ( P, V, Receiver).\n\nThe [[Set]] internal method needs to check the receiver argument.\n\n\nTest case:\n---\n(function(y){ Object.create(arguments)[0] = 2; return y})(1)\n---\n\nExpected: Returns 1\nActual: Returns 2"},{"commentid":7714,"comment_count":1,"who":{"uid":"allen","name":"Allen Wirfs-Brock"},"bug_when":"2014-04-12 16:53:26 -0700","thetext":"fixed in rev24 editor's draft"},{"commentid":7985,"comment_count":2,"who":{"uid":"allen","name":"Allen Wirfs-Brock"},"bug_when":"2014-04-29 21:52:38 -0700","thetext":"fixed in rev24"}]}}
---
